msteams-platform/concepts/deploy-and-publish/add-default-install-scope.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Add a default install scope and group capability
+description: Describes how to add default install scope and group capability for apps.
+ms.topic: how-to
+ms.author: surbhigupta
+---
+
+
+# Add a default install scope and group capability
```

Comment:
@KirtiPereira I think this topic needs some work. From what I can tell, there are two app configuration tasks here that need steps. Here is how we could possibly structure it:

## Configure app's default install scope

Explain what this configuration is for, why you would use it, and how to configure `defaultInstallScope` in the manifest (we should provide a JSON example for context).

## Configure the default capability for a shared scopes

Same as above but this time for `defaultGroupCapability`.

Reply:
@heath-hamilton - I am working on this suggestion.